From 24471e0f4b3f31e17f4cbbddc9333b8a65eaee41 Mon Sep 17 00:00:00 2001
From: Fei Yang <fei.yang@intel.com>
Date: Tue, 22 May 2018 14:20:21 -0700
Subject: [PATCH 1480/2302] drm/i915: fix compile warning

drivers/gpu/drm/i915/intel_initial_modeset.c: In function 'update_primary_plane_state':
warning: format '%ld' expects argument of type 'long int', but argument 2 has type 'size_t
printk("Splash size %ld, palette size %d\n", splash_info->fw->size, palette_size);

Signed-off-by: Fei Yang <fei.yang@intel.com>
---
 drivers/gpu/drm/i915/intel_initial_modeset.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_initial_modeset.c b/drivers/gpu/drm/i915/intel_initial_modeset.c
index 493b401..9861824 100644
--- a/drivers/gpu/drm/i915/intel_initial_modeset.c
+++ b/drivers/gpu/drm/i915/intel_initial_modeset.c
@@ -432,7 +432,7 @@ static int update_primary_plane_state(struct drm_atomic_state *state,
 
 	/* Color palette is appended after image data, it uses 24-bits for each index entry */
 	palette_size = (splash_info->fw->size - (splash_info->width * splash_info->height)) / 3;
-	printk("Splash size %ld, palette size %d\n", splash_info->fw->size, palette_size);
+	printk("Splash size %zu, palette size %u\n", splash_info->fw->size, palette_size);
 	if (0 == palette_size) {
 		DRM_ERROR("Splash image does not contain color palette data\n");
 		return -1;
-- 
2.7.4

